webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -90,6 +90,17 @@ def timestamp_to_date(
 
 
 @library.filter
+def time_ago(dt, format='%a, %b %d %H:%M %Z'):
+    return jinja2.Markup(
+        '<time datetime="{}" class="ago">{}</time>'
+        .format(
+            dt.isoformat(),
+            dt.strftime(format)
+        )
+    )
+
+
```

Comment:
Ah, I haven't checked that. I assumed we had some somewhere deep in the site, but maybe it is not used anymore.